include/range/v3/at.hpp

```diff
@@ -35,10 +36,15 @@ namespace ranges
             auto operator()(Rng &&rng, range_difference_type_t<Rng> n) const ->
                 decltype(begin(rng)[n])
             {
+                if (SizedRange<Rng>()) {
+                    RANGES_EXPECT(n < ranges::distance(rng));
```

Comment:
We've made you change this like ten times already, but I observe this whole conditional could be:
```c++
RANGES_EXPECT(!SizedRange<Rng>() || n < ranges::distance(rng));
```
I'll let you decide if you want to change it again ;)